src/bin/hexyl.rs

```diff
@@ -149,6 +149,17 @@ fn run() -> Result<(), AnyhowError> {
                     A negative value is valid and calculates an offset relative to the \
                     end of the file.",
                 ),
+        )
+        .arg(
+            Arg::new("columns")
+                .short('w')
+                .long("columns")
+                .takes_value(true)
+                .value_name("N")
+                .help(
+                    "Sets the number of hex data columns to be displayed. \
+                    Cannot be used with other width-setting options.",
```

Comment:
Should this be omitted for now?

Reply:
I already have the pull request for terminal width ready, I'll commit it if there's no other problems. I just wanted to make sure that the major architectural changes were correct.

Reply:
Ok, in that case, yes. Please add it to this PR. Or leave the help text as-is, and we do a second PR with the auto-width changes.